dev/EnvironmentManager/ChangeTracker/EnvironmentVariableChangeTracker.cpp

```diff
@@ -2,37 +2,14 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 #include <pch.h>
 #include "EnvironmentVariableChangeTracker.h"
+#include "EnvironmentVariableChangeTrackerHelper.h"
 
 namespace winrt::Microsoft::Windows::System::implementation
 {
-    EnvironmentVariableChangeTracker::EnvironmentVariableChangeTracker(const std::wstring& key, const std::wstring& valueToSet, EnvironmentManager::Scope scope)
+    EnvironmentVariableChangeTracker::EnvironmentVariableChangeTracker(const std::wstring& evName, const std::wstring& valueToSet, EnvironmentManager::Scope scope)
+        : m_eVName(evName), m_eVValue(valueToSet), m_Scope(scope)
```

Comment:
NIT: 1 line? Ick

May just be me and not coding guidelines but I prefer 1 attribute per line. And yes, I know that may sound contradictory to my stance on function parameters, but they're different -- you set a breakpoint on a function, not each parameter. OTOH if these attributes are objects how do you set a breakpoint to trace into one constructor?

Reply:
What do you mean by 1 attribute per line?  The input parameters 1 per line?